serverless/src/shared/traverseGraph.js

```diff
@@ -0,0 +1,52 @@
+import { cloneDeep } from 'lodash'
+
+import formatCsvPath from '@/shared/formatCsvPath'
+import getCsvLongNameFlag from '@/shared/getCsvLongNameFlag'
+import getCsvProviderUrlFlag from '@/shared/getCsvProviderUrlFlag'
+import getNarrowers from '@/shared/getNarrowers'
+
+const traverseGraph = async (csvHeadersCount, providerUrlsMap, longNamesMap, scheme, n, map, path = [], paths = []) => {
```

Comment:
maybe call this buildHierarchalCsvPaths to be a bit more descriptive

Reply:
Done